stream-chat-android-client/src/main/res/drawable/stream_ic_notification.xml

```diff
@@ -1,8 +1,11 @@
-<shape
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        android:shape="rectangle">
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle"
+    >
```

Comment:
This will be controversial, but it does wonders for VCS history in the long run 👍

Reply:
In "smaller xml tags" could see a bit weird, but it will help us a lot on large xmls